V/Variography/Compat.toml

```diff
@@ -1,23 +1,23 @@
-[0]
-Distances = "0.5-0"
-Parameters = "0.7.3-0"
-RecipesBase = "0.2.3-0"
-SpecialFunctions = "0.3.8-0"
-
 ["0-0.1"]
 GeoStatsBase = "0.0.5-0"
 julia = "0.6-0"
 
-["0.0.3-0"]
-StaticArrays = "0.7-0"
+["0-0.3.6"]
+Distances = "0.5-0"
+Parameters = "0.7.3-0"
+RecipesBase = "0.2.3-0"
+SpecialFunctions = "0.3.8-0"
```

Comment:
Please keep these for 0.3.7

Reply:
These lower bounds on the dependencies are useless. They were made long ago before Julia 1.x era. I want to clean up my Project.toml in Julia 1.x and only add lower bounds again if strictly needed. Can you please go ahead and merge this?

Reply:
So you are saying that this package version suddenly is compatible with Distances 0.1, 0.2, 0.3 etc?